matchms/filtering/add_precursor_mz.py

```diff
@@ -1,16 +1,32 @@
 import logging
 from matchms.utils import get_first_common_element
-from ..typing import SpectrumType
```

Comment:
What was wrong with importing the SpectrumType and having the type hints?

Reply:
Might work now, I'll have to check. Before I got an issue because of a loop: typing (imports Spectrum) --> add_precursor_mz (imports typing) --> Spectrum (imports add_precursor_mz)